docs/website/docs/index.md

```diff
@@ -23,7 +23,7 @@ IREE supports importing from a variety of ML frameworks:
 - [x] TensorFlow
 - [x] TensorFlow Lite
 - [x] JAX
-- [ ] PyTorch (planned)
+- [x] PyTorch
```

Comment:
Do we have sample scripts, Python notebooks, etc. showing how to use PyTorch with IREE?

I'd love to add some material to these places (and others):
* https://iree-org.github.io/iree/community/
* https://iree-org.github.io/iree/getting-started/
* https://github.com/iree-org/iree/tree/main/samples/colab

Reply:
We do have a directory in the iree-samples repo: https://github.com/iree-org/iree-samples/tree/main/iree-torch ... it's pretty empty though :)

Reply:
We have a couple here: https://github.com/iree-org/iree-torch/tree/main/examples.  They're both a bert example -- one's just in a colab format.  There are also a number of examples in this repo: https://github.com/nod-ai/SHARK

The one thing I have to think about is how to avoid just copying and pasting the examples across repos / parts of individual repos.  For example, I'd like to have some parts of [this notebook](https://github.com/iree-org/iree-torch/blob/main/examples/bert.ipynb) in the readme for that repo, and I'd also like to be able to automatically run it for testing purposes.  Creating a copy that's automatically run as well as copying to the readme in that repo and then copying to the `iree-org/iree` repo seems like a pretty obviously bad solution.

Any suggestions you may have are welcome.